include/ipc-client.h

```diff
@@ -24,6 +24,10 @@ char *get_socketpath(void);
  */
 int ipc_open_socket(const char *socket_path);
 /**
+ * Issues a single IPC command without reading response.
+ */
+void ipc_single_command_no_response(int socketfd, uint32_t type, const char *payload, uint32_t *len);
```

Comment:
Don't we need to read the response for further reads to make sense? Also, if chaning the client fixes this then I'm skeptical of the fix, malicious IPC clients shouldn't be able to crash sway.